spec/lib/presenters/show_presenter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+require "presenters/show_presenter"
+
+RSpec.describe ShowPresenter do
+  describe "#page_title" do
+    it "is the resource's key attribute" do
+      customer = build(:customer, name: "Worf")
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      expect(presenter.page_title).to eq("Worf")
+    end
+  end
+
+  describe "#attribute_names" do
+    it "uses attribute names from the dashboard" do
+      customer = build(:customer, name: "Worf")
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      expect(presenter.attribute_names).to eq([:email, :lifetime_value])
+    end
+  end
+
+  describe "#render_attribute" do
+    it "renders the attribute" do
+      email = "worf@enterprise.uss"
+      customer = build(:customer, email: email)
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      rendered = presenter.render_attribute(:email)
+
+      expect(rendered).to eq(email)
+    end
+  end
+
+  describe "#edit_path" do
+    it "returns the edit path for the displayed resource" do
+      customer = create(:customer)
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      edit_path = presenter.edit_path
+
+      expect(edit_path).to eq("/customers/#{customer.id}/edit")
```

Comment:
Is it possible to use `edit_customer_path(customer)` instead of hard-coding the string?